schemas/log.yml

```diff
@@ -43,3 +43,28 @@
       short: Name of the logger.
       description: >
         The name of the logger inside an application. This is usually the name of the class which initialized the logger, or can be a custom name. 
+
+    - name: origin.file.name
+      level: core
```

Comment:
Could we make all these extended? Sorry, should have spotted this earlier.

Reply:
sure, np 🙂